docs/users/user-guide.md

```diff
@@ -2,15 +2,59 @@
 
 ## Finding projects
 
+Published projects can be browsed at [https://biosimulations.org/projects](https://biosimulations.org/projects). Each card presents a project, with a thumbnail, and title. Mousing over the thumbnail shows additional details about the project. You can customize the attributes you want to view. 
```

Comment:
Remove extra second comma in "each card presents a project, with a thumbnail, and title"